storage/innobase/handler/handler0alter.cc

```diff
@@ -7117,13 +7089,13 @@ Rename a given index in the InnoDB data dictionary.
 @retval false Success */
 static MY_ATTRIBUTE((warn_unused_result))
 bool
-rename_index_in_data_dictionary(
+innobase_rename_index_in_data_dictionary(
 /*============================*/
 	const dict_index_t*	index,
 	const char*		new_name,
 	trx_t*			trx)
 {
-	DBUG_ENTER("rename_index_in_data_dictionary");
+	DBUG_ENTER("innobase_rename_index_in_data_dictionary");
```

Comment:
I would rename this to `rename_index_try()` to be in line with the other function names.
That should also allow all arguments to be listed in a single line, to be more `grep` friendly:
```c++
MY_ATTRIBUTE((warn_unused_result))
static bool
rename_index_try(const dict_index_t* index, const char* new_name, trx_t* trx)
```